src/ripple/app/tx/impl/PayChan.cpp

```diff
@@ -37,86 +37,88 @@ namespace ripple {
 /*
     PaymentChannel
 
-        Payment channels permit off-ledger checkpoints of XRP payments flowing
-        in a single direction. A channel sequesters the owner's XRP in its own
-        ledger entry. The owner can authorize the recipient to claim up to a
-        given balance by giving the receiver a signed message (off-ledger). The
-        recipient can use this signed message to claim any unpaid balance while
-        the channel remains open. The owner can top off the line as needed. If
-        the channel has not paid out all its funds, the owner must wait out a
-        delay to close the channel to give the recipient a chance to supply any
-        claims. The recipient can close the channel at any time. Any transaction
-        that touches the channel after the expiration time will close the
-        channel. The total amount paid increases monotonically as newer claims
-        are issued. When the channel is closed any remaining balance is returned
-        to the owner. Channels are intended to permit intermittent off-ledger
-        settlement of ILP trust lines as balances get substantial. For
-        bidirectional channels, a payment channel can be used in each direction.
+        Payment channels permit off-ledger XRP payments flowing between two
+        accounts in both directions. A channel sequesters the participants' XRP
+        in its own ledger entry. The participants can authorize the counterparty
+        to claim up to a given amount by giving the receiver a signed message
+        (off-ledger). The recipient can use this signed message to settle the
+        channel balance. A participant can authorize a payment amount exceeding
+        their channel balance as long as the difference between the two sides'
+        authorized payments does not exceed the respective channel balance.
+        Payment amounts can be netted by subtracting both by the lower amount
+        and incrementing the sequence number on the authorized payments (off-
+        ledger).
+        Either participant can top off their channel balance as needed. When
+        submitting a channel claim, the participant must wait out a delay to
+        close the channel to give the counterparty a chance to supply any
+        additional claims. Any transaction that touches the channel after the
+        expiration time will close the channel. When the channel is closed any
+        remaining balance is returned to the owners.
 
     PaymentChannelCreate
 
-        Create a unidirectional channel. The parameters are:
+        Create a bidirectional XRP payment channel. The parameters are:
         Destination
-            The recipient at the end of the channel.
+            The other channel participant.
```

Comment:
Participants are symmetric (other than the owner count). I'm not sure we should call one of them "destination". We should think of a better term.